pac4j-core/src/main/java/org/pac4j/core/credentials/extractor/BearerAuthExtractor.java

```diff
@@ -0,0 +1,29 @@
+package org.pac4j.core.credentials.extractor;
+
+import org.pac4j.core.context.HttpConstants;
+import org.pac4j.core.context.WebContext;
+import org.pac4j.core.credentials.TokenCredentials;
+
+/**
+ * To extract an RFC 6750 bearer auth header.
+ *
+ * @author Graham Leggett
+ * @since 4.0.1
```

Comment:
It's 4.0.0

Reply:
Updated it to 3.5.0 - unless it needs to be more recent?

Reply:
Done.

Reply:
3.5.0 is OK as you submitted a dedicated pull request.